Content.Server/GameObjects/Components/Weapon/Melee/StunbatonComponent.cs

```diff
@@ -93,7 +93,7 @@ protected override bool OnHitEntities(IReadOnlyList<IEntity> entities, AttackEve
             if (!Cell.TryUseCharge(EnergyPerUse))
                 return true;
 
-            EntitySystem.Get<AudioSystem>().PlayAtCoords("/Audio/Weapons/egloves.ogg", Owner.Transform.GridPosition, AudioHelpers.WithVariation(0.25f));
+            EntitySystem.Get<AudioSystem>().PlayAtCoords("/Audio/Weapons/Egloves.ogg", Owner.Transform.GridPosition, AudioHelpers.WithVariation(0.25f));
```

Comment:
It would be better to leave this as it was before and simply rename the file to match the case

Reply:
Ah I see, will do.